pandas/io/excel/_base.py

```diff
@@ -1100,11 +1130,10 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         self.close()
 
-    def close(self):
+    def close(self) -> None:
         """synonym for save, to make it more file-like"""
-        content = self.save()
+        self.save()
         self.handles.close()
-        return content
```

Comment:
Not sure whether we can remove the return value (looks like a public function) on the other side ["None of the methods and properties are considered public."](https://pandas.pydata.org/docs/reference/api/pandas.ExcelWriter.html). I favor removing the return value.

Reply:
Content is always None, this is just for clarity, does not make a difference